final_prep/mod_zero_hero.rb

```diff
@@ -1,52 +1,113 @@
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
+greeting = "Now introducing #{hero_name}!"
+puts greeting
+catchphrase = "Among immunity to fire, he also has #{special_ability}."
+puts catchphrase
 
 # Declare two variables - power AND energy - set to integers
+power = 100
+energy = 200
 
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
+full_power = power * 500
+full_energy = energy + 150
+puts full_power
+puts full_energy
 
 # Declare two variables - is_human and identity_concealed - assigned to booleans
-
+is_human = false
+identity_concealed = false
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
+arch_enemies = ["Rasputin", "Hecate", "Baba Yaga"]
+sidekicks = ["Abe Sapien", "Sarah Hughes", "Azzael"]
 
 # Print the first sidekick to your terminal
-
+puts sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+puts arch_enemies[-1]
 # Write some code to add a new arch_enemy to the arch_enemies array
-
+arch_enemies << "Gruagach"
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
-
+puts arch_enemies
 # Remove the first sidekick from the sidekicks array
-
+sidekicks.delete_at(0)
 # Print the sidekicks array to terminal to ensure you added a new sidekick
-
+puts sidekicks
 # Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
 #   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
+#   - save_the_day should be a string a hero would say once they save the day
 #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
+def assess_situation(danger_level, save_the_day, bad_excuse)
+  puts "The danger level is #{danger_level}."
+  puts "I saved the day and they #{save_the_day}."
+  puts "Whoops I forgot to #{bad_excuse}"
+end
+
+
 
 # Your function should include an if/else statement that meets the following criteria
 #   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the bad_excuse to the terminal
 #   - Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal
 #   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
+def assess_situation(danger_level)
+
+  save_the_day = "Didn't even buy me a drink."
+  bad_excuse = "Forgot to feed my cats."
+
+  if danger_level > 50
+  puts "#{bad_excuse}"
+
+  elsif danger_level >= 10
```

Comment:
While this works because order in conditionals matter, it is not best practice to have 2 conditions that would be truthy for values... in this case, numbers greater than 50 are also greater than 10. You can think about keeping more complex logic in your `else` block.